### PR TITLE
Add show and head method to DataCollection

### PR DESCRIPTION
**Pull request by GuoRentong**
#927

### CODE REVIEW

File: towhee/functional/data_collection.py [hunk 1]
```diff
@@ -727,6 +728,21 @@ def inner():
 
         return self.factory(inner())
 
+    def head(self, n: int = 5):
```
Comment (reiase) [inline towhee/functional/data_collection.py:731]: please add some code example for user API

File: tests/unittests/functional/test_mixins.py [hunk 2]
```diff
@@ -26,3 +33,32 @@
...
+        logo_path = os.path.join(Path(__file__).parent.parent.parent.parent.resolve(), 'towhee_logo.png')
+        img = cv2.imread(logo_path)
+        towhee_img = Image(img, 'BGR')
+        dc = DataCollection([[1, img, towhee_img], [2, img, towhee_img]])
+        to_printable_table(dc._iterable, tablefmt='plain')
+        to_printable_table(dc._iterable, tablefmt='html')
+
+    def test_show(self):
+        dc = DataCollection([[1.1, 2.2], [3.3, 4.4], [5.5, 6.6]])
+        dc.show(tablefmt='plain')
```
Comment (reiase) [inline tests/unittests/functional/test_mixins.py:63]: there should be some asserts that check for expected outputs

**Review by reiase — CHANGES_REQUESTED** [review]

File: towhee/functional/data_collection.py (hunk 1 above)
Comment (GuoRentong) [reply towhee/functional/data_collection.py:731]: > 

added a simple example

**Pull request merged**